components/GitHubEventAction.tsx

```diff
@@ -86,19 +102,18 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       );
     }
     case 'PullRequestReviewEvent': {
-      const action = payload?.action;
-      const prNum = payload?.pull_request?.number;
-      const prURL = payload?.pull_request?.html_url;
+      const action = (payload as PullRequestReviewEvent).action;
+      const prNum = (payload as PullRequestReviewEvent).pull_request.number;
+      const prURL = (payload as PullRequestReviewEvent).pull_request.html_url;
       if (!action || !prNum || !prURL) {
         return unknown;
       }
-      const actionStr = action === 'created' ? 'reviewed' : action;
+      const actionStr = 'reviewed';
```

Comment:
Given what you told me about 
```
Error 1. This condition will always return 'false' since the types '"edited" | "dismissed" | "submitted"' and '"created"' have no overlap.   
``` 

Maybe `submitted` is what you're looking for?

```suggestion
      const actionStr = action === 'submitted' ? 'reviewed' : action;```